test/__tests__/launchdarkly_audience-cdk.test.ts

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs';
+import path from 'path';
+import { processCdkV2Workflow } from '../../src/cdk/v2/handler';
+import tags from '../../src/v0/util/tags';
+
+const integration = 'launchdarkly_audience';
+const destName = 'LaunchDarkly Audience';
+
```

Comment:
@koladilip can't we make cdk test suite as generic one, similar to component.test.ts, to avoid writing for each destType ?

Reply:
@Gauravudia can you get this test case driven via existing component.test.ts itself by adding `cdkV2Enabled`: `TRUE` in the config ?

Reply:
We are mocking batch size config in `launchdarkly_audience-cdk.test.ts` file. Which is not possible in `component.test.ts`  file. Since it is common for all the integrations.